diff --git a/dep/g3dlite/include/G3D/Array.h b/dep/g3dlite/include/G3D/Array.h
index 788237dce..f4387ad5a 100644
--- a/dep/g3dlite/include/G3D/Array.h
+++ b/dep/g3dlite/include/G3D/Array.h
@@ -346,7 +346,11 @@ public:
 
    /** Resizes this to match the size of \a other and then copies the data from other using memcpy.  This is only safe for POD types */
    void copyPOD(const Array<T>& other) {
+#if __cplusplus >= 201703L
        static_assert(std::is_standard_layout_v<T> && std::is_trivial_v<T>, "copyPOD called on non-POD type");
+#else
+       static_assert(std::is_standard_layout<T>::value && std::is_trivial<T>::value, "copyPOD called on non-POD type");
+#endif
        if (numAllocated < other.num) {
            m_memoryManager->free(data);
            data = NULL;
@@ -365,7 +369,11 @@ public:
    /** Resizes this to just barely match the size of \a other + itself and then copies the data to the end of the array from other using memcpy.  
         This is only safe for POD types */
    void appendPOD(const Array<T>& other) {
+#if __cplusplus >= 201703L
        static_assert(std::is_standard_layout_v<T> && std::is_trivial_v<T>, "appendPOD called on non-POD type");
+#else
+       static_assert(std::is_standard_layout<T>::value && std::is_trivial<T>::value, "appendPOD called on non-POD type");
+#endif
        const size_t oldSize = num;
        num += other.num;
        if (numAllocated < num) {
