diff --git a/dep/g3dlite/include/G3D/Ray.h b/dep/g3dlite/include/G3D/Ray.h
index 79afc7616..0449e549d 100644
--- a/dep/g3dlite/include/G3D/Ray.h
+++ b/dep/g3dlite/include/G3D/Ray.h
@@ -33,22 +33,6 @@ class Ray {
     /** 1.0 / direction */
     Vector3 m_invDirection;
 
-    
-    /** The following are for the "ray slope" optimization from
-      "Fast Ray / Axis-Aligned Bounding Box Overlap Tests using Ray Slopes" 
-      by Martin Eisemann, Thorsten Grosch, Stefan Müller and Marcus Magnor
-      Computer Graphics Lab, TU Braunschweig, Germany and
-      University of Koblenz-Landau, Germany */
-    enum Classification {MMM, MMP, MPM, MPP, PMM, PMP, PPM, PPP, POO, MOO, OPO, OMO, OOP, OOM, OMM, OMP, OPM, OPP, MOM, MOP, POM, POP, MMO, MPO, PMO, PPO};    
-
-    Classification classification;
-
-    /** ray slope */
-    float ibyj, jbyi, kbyj, jbyk, ibyk, kbyi;
-
-    /** Precomputed components */
-    float c_xy, c_xz, c_yx, c_yz, c_zx, c_zy;
-
 public:
     /** \param direction Assumed to have unit length */
     void set(const Point3& origin, const Vector3& direction);
diff --git a/dep/g3dlite/source/Ray.cpp b/dep/g3dlite/source/Ray.cpp
index 8a8cf77c5..c2a2f4ff6 100644
--- a/dep/g3dlite/source/Ray.cpp
+++ b/dep/g3dlite/source/Ray.cpp
@@ -21,105 +21,6 @@ void Ray::set(const Vector3& origin, const Vector3& direction) {
     debugAssert(direction.isUnit());
     
     m_invDirection = Vector3::one() / direction;
-    
-    // ray slope
-    ibyj = m_direction.x * m_invDirection.y;
-    jbyi = m_direction.y * m_invDirection.x;
-    jbyk = m_direction.y * m_invDirection.z;
-    kbyj = m_direction.z * m_invDirection.y;
-    ibyk = m_direction.x * m_invDirection.z;
-    kbyi = m_direction.z * m_invDirection.x;
-    
-    // precomputed terms
-    c_xy = m_origin.y - jbyi * m_origin.x;
-    c_xz = m_origin.z - kbyi * m_origin.x;
-    c_yx = m_origin.x - ibyj * m_origin.y;
-    c_yz = m_origin.z - kbyj * m_origin.y;
-    c_zx = m_origin.x - ibyk * m_origin.z;
-    c_zy = m_origin.y - jbyk * m_origin.z;    
-    
-    //ray slope classification
-    if (m_direction.x < 0) {
-        if (m_direction.y < 0) {
-            if (m_direction.z < 0) {
-                classification = MMM;
-            } else if (m_direction.z > 0) {
-                classification = MMP;
-            } else { //(m_direction.z >= 0)
-                classification = MMO;
-            }
-        } else { //(m_direction.y >= 0)
-            if (m_direction.z < 0) {
-                if (m_direction.y == 0) {
-                    classification = MOM;
-                } else {
-                    classification = MPM;
-                }
-            } else { //(m_direction.z >= 0)
-                if ((m_direction.y == 0) && (m_direction.z == 0)) {
-                    classification = MOO;
-                } else if (m_direction.z == 0) {
-                    classification = MPO;
-                } else if (m_direction.y == 0) {
-                    classification = MOP;
-                } else {
-                    classification = MPP;
-                }
-            }
-        }
-    } else { //(m_direction.x >= 0)
-        if (m_direction.y < 0) {
-            if (m_direction.z < 0) {
-                if (m_direction.x == 0) {
-                    classification = OMM;
-                } else {
-                    classification = PMM;
-                }
-            } else { //(m_direction.z >= 0)
-                if ((m_direction.x == 0) && (m_direction.z == 0)) {
-                    classification = OMO;
-                } else if (m_direction.z == 0) {
-                    classification = PMO;
-                } else if (m_direction.x == 0) {
-                    classification = OMP;
-                } else {
-                    classification = PMP;
-                }
-            }
-        } else { //(m_direction.y >= 0)
-            if (m_direction.z < 0) {
-                if ((m_direction.x == 0) && (m_direction.y == 0)) {
-                    classification = OOM;
-                } else if (m_direction.x == 0) {
-                    classification = OPM;
-                } else if (m_direction.y == 0) {
-                    classification = POM;
-                } else {
-                    classification = PPM;
-                }
-            } else { //(m_direction.z > 0)
-                if (m_direction.x == 0) {
-                    if (m_direction.y == 0) {
-                        classification = OOP;
-                    } else if (m_direction.z == 0) {
-                        classification = OPO;
-                    } else {
-                        classification = OPP;
-                    }
-                } else {
-                    if ((m_direction.y == 0) && (m_direction.z == 0)) {
-                        classification = POO;
-                    } else if (m_direction.y == 0) {
-                        classification = POP;
-                    } else if (m_direction.z == 0) {
-                        classification = PPO;
-                    } else {
-                        classification = PPP;
-                    }
-                }
-            }            
-        }
-    }
 }
 
 
